Sources/TuistCore/Models/Project.swift

```diff
@@ -32,7 +32,7 @@ public class Project: Equatable, CustomStringConvertible {
     public let fileName: String
 
     /// Project targets.
-    public private(set) var targets: [Target]
+    public var targets: [Target]
```

Comment:
I'd discourage public mutations of the models and rather expose explicit methods to do the mutations. This will ease the work of moving to a value-typed graph.

Reply:
Agreed, I was not sure how to tackle this, a method is much nicer, thanks for the suggestion!